python/tank/deploy/tank_commands/setup_project_params.py

```diff
@@ -1042,10 +1047,10 @@ def _process_config(self, config_uri):
                 # either a folder or zip file!
                 if config_uri.endswith(".zip"):
                     self._log.info("Hang on, unzipping configuration...")
-                    return (self._process_config_zip(config_uri), "local")
+                    return (self._process_config_zip(config_uri), "zip")
                 else:
                     self._log.info("Hang on, loading configuration...")
-                    return (self._process_config_dir(config_uri), "local")
+                    return (self._process_config_dir(config_uri), self._PROJECT_CONFIG)
```

Comment:
I made a constant because it is being used in multiple places, so it felt safer to have a constant. The other strings however are single use so making constants is less important imho.